recipes/broh5/meta.yaml

```diff
@@ -30,10 +30,6 @@ requirements:
 test:
   imports:
     - broh5
-  commands:
```

Comment:
Comment it out instead of removing.  We should try to re-enable it when the downstream issue is resolved. pip-check helps us find problems in the stack.